tools/sandbox/application.py

```diff
@@ -175,7 +175,6 @@ def analyze(self) -> None:
                     )
                     LOG.debug("\n".join(model_verification_errors))
                 LOG.debug(line)
```

Comment:
Pre-existing: This is never reached if we go through line 165 `continue`. I would put this at the beginning of the if/else chain, to prevent this type of confusion.
I would also remove that `continue`.

Reply:
Reordered and removed `continue` 😄